core/examples/luigi-sample-angular/README.md

```diff
@@ -1,25 +1,51 @@
-# Sample luigi sample application written in Angular
+# Sample luigi application written in Angular
 
 ## Development
 
-This mono repository uses [Lerna](https://lernajs.io/) for project management. Ensure that you have installed lerna, if not then run   
-`npm install --global lerna`
+Linking and building with [Lerna](https://lernajs.io/) is the preferred development method. This monorepo uses Lerna for project management. 
 
-To start developing this application, follow these steps:
+To have this application running, follow these steps:
 
-1. Install dependencies.  
-`lerna bootstrap` executes npm install and links cross-dependencies
-```bash
-lerna bootstrap
-```
+1. Install Lerna globally.
+    ```bash
+    npm install -g lerna
+    ```
 
-2. Bundle cross-dependencies.  
-`lerna run` runs the given script in every package referenced in lerna.json
-```bash
-lerna run bundle
-```
+2. Install dependencies.
+    ```bash
+    # Lerna bootstrap makes npm install and links cross-dependencies.
 
-## Run server
+    lerna bootstrap
+    ```
+
+3. Bundle luigi core.
+    ```bash
+    # Lerna runs the bundle script in every package where the script exists.
+
+    lerna run bundle
+    ```
+
+4. Start the example application.
+    ```bash
+    npm start
+    ```
+
+5. Open it in your browser.
+
+    In your favourite browser, go to http://localhost:4200/
+
+6. Reflect changes on luigi code on the example app.
+
+    a. For Luigi Core you have to let it bundle again on every change you apply to it. The easiest approach is to open Luigi´s root folder in another tab in your terminal window and have the following running: 
+    ```bash    
+    cd core && npm run bundle -- --watch
+    ```
+    b. Luigi Client is not bundled, so you are fine just updating it without bundling. 
+    
+    c. *The autoreload of your application only updates the application*. Type CMD + R to reflect the changes of the linked modules (Luigi Core and Luigi Client) on the website.
```

Comment:
We do not use italics in Kyma documentation.

``The auto-reload of your application only updates the application. Type `CMD + R` to reflect the changes of the linked modules (Luigi Core and Luigi Client) in the website.``

Reply:
done